Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,47 @@
+| Support downgrade                                                  | P0       | etcd will support downgrade starting from 3.6.0. But it will also support offline downgrade from 3.5 to 3.4. |
+| StoreV2 deprecation                                                | P0       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Evaluate and (Gradulate or deprecate/remove) experimental features | P2       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Support /livez and /readyz endpoints                               | P1       | It provides clearer APIs, and can also workaround the stalled writes issue                                   |
+| Bump gRPC                                                          | P0       | It isn't guaranteed to be resolved in 3.6, and might be postponed to 3.7 depending on the effort and risk.   |
+| Deprecate grpc-gateway or bump it                                  | P1       | It isn't guaranteed to be resolved in 3.6, and might be postponed to 3.7 depending on the effort and risk.   |
+| Release raft 3.6.0                                                 | P1       | etcd 3.6.0 will depends on raft 3.6.0                                                                        |
+| bbolt: Add logger into bbolt                                       | P1       | It's important to diagnose bbolt issues                                                                      |
+| bbolt: Add surgery commands                                        | P1       | Surgery commands are important for fixing corrupted db files                                                 |
+| Release bbolt 1.3.8                                                | P1       | etcd 3.6.0 will depends on bbolt 1.3.8                                                                       |
+
+## v3.7.0
+
+For a full list of tasks in v3.7.0, please see [milestone etcd-v3.7](https://github.com/etcd-io/etcd/milestone/39).
+
+| Title                                                             | Priority | Note                                                                              |
+|-------------------------------------------------------------------|----------|-----------------------------------------------------------------------------------|
+| StoreV2 deprecation                                               | P0       | Finish the remaining tasks 3.7.                                                   |
+| Evaluate and (graduate or deprecate/remove) experimental features | P2       | Finish the remaining tasks 3.7.                                                   |
+| Refactor lease: Lease might be revoked by mistake by old leader   | P0       | to be investigated & discussed                                                    |
```

Comment:
I would be careful with adding P0 for next milestone. Taking a blocker for the next release without any prior setup might negatively affect delivery time. I would suggest to move P0 for v3.7.0 to P1 for v3.6.0

Reply:
Updated